timemachine/fe/free_energy.py

```diff
@@ -803,6 +803,29 @@ def make_pair_bar_plots(res: PairBarResult, temperature: float, prefix: str) ->
     return PairBarPlots(dG_errs_png, overlap_summary_png, overlap_detail_png)
 
 
+def assert_potentials_compatible(bps1: Sequence[BoundPotential], bps2: Sequence[BoundPotential]):
+    """Asserts that two sequences of bound potentials are equivalent except for their parameters"""
+
+    def field_equal(x1, x2):
+        assert type(x1) is type(x2)
```

Comment:
Could you add string messages to these assertions? Be helpful to know what the error is immediately without having to find the traceback (which we don't always have)

Reply:
Would also strengthen tests to verify that the error you expect is being exercised

Reply:
Improved assertion failure messages and strengthened tests in https://github.com/proteneer/timemachine/pull/1366/commits/ba6fb80cb2c9db0b658520627724777d7efc4e05